docs/includes/topic_f5lbaas-l7_content_switching.rst

```diff
@@ -0,0 +1,227 @@
+
+Policy Rule Comparison Type:
+
+    In addition to a rule type there are five comparison types that are applied to the rule’s operand and compared with the rule value to determine if a match exists.
+
+        STARTS_WITH – operand starts with string
+        ENDS_WITH – operand ends with string
+        EQUAL_TO – operand matches the string
+        CONTAINS – operand contains a substring value
+        REGEX – operand matches the provided regular expression. **(currently not supported)**
+
+Neutron Policy Logic:
+
+All the rules in an L7 policy must match before the associated action is executed. So if a policy has a set of rules: R1, R2, … Rn, and an action, A, then the following logic holds:
+
+    If (R1 and R2 and … Rn ) then A
+
+Policy rules can also be negated by using the –invert parameter when specifying the rules. For example, the comparison type, EQUAL_TO can be transformed to NOT_EQUAL_TO, by specifying –invert.
+
+L7 policies are ranked by a position value and are evaluated according to their rank. The first policy that evaluates to true is executed and all subsequent policies are skipped. Given a set of n policies, where policy Pn has a rank n and an action An, the following logic holds:
+
+    If (P1) then A1
+    Else if (P2) then A2
+    …
+    Else if (Pn) then An
+    Else:
+        Send request to default pool
+
+OpenStack Policy/Rules Definition Versus BIG-IP® Policy/Rules:
+``````````````````````````````````````````````````````````````
+
+The Neutron L7 terminology does not directly align with the common vocabulary of BIG-IP Local Traffic Manager. In the BIG-IP LTM, policies also have a set of rules, but it is the rules that specify actions and not the policy. Also, policies attached to a virtual server on the BIG-IP are all evaluated regardless of the truth of the associated rules. In addition to this difference the BIG-IP policies have no ordinal, it is the BIG-IP rules that have this attribute. Because of these confusing differences it is useful to attempt to define the terms as they apply to each domain.
+
+    +------------------+-------------------------------+
+    | Neutron LBaaS L7 | BIG-IP® Local Traffic Manager |
+    ===================+===============================+
+    | Policy           | Policy Rules (wrapper_policy) |
+    +------------------+-------------------------------+
+    | Policy Action    | Rule Action                   |
+    +------------------+-------------------------------+
+    | Policy Position  | Rule Ordinal                  |
+    +------------------+-------------------------------+
+    | Rule             | Rule Conditions               |
+    +------------------+-------------------------------+
+
+The BIG-IP LTM policy has a name, description, a set of rules, and a strategy on how those rule should be evaluated. In fact, L7 policies in OpenStack are more akin to a collection BIG-IP LTM policy rules that are evaluated with the ‘First match’ strategy.
```

Comment:
on how those rule should be evaluated

should be:

on how those rules are evaluated